src/webhook/index.ts

```diff
@@ -88,7 +97,11 @@ const singleUserHandler = async (
       result = {
         context: { data: data },
         replies: [
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
           createGreetingMessage(),
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
```

Comment:
As [discussed on Slack](https://g0v-slack-archive.g0v.ronny.tw/index/channel/C2PPMRQGP/2023-08#ts-1692067805.6942) we decided to create a new ticket for typing `handlers/tutorial` and removing ts-ignore here.